l4_iteration/homework/hw4_cc_iteration.py

```diff
@@ -0,0 +1,90 @@
+CURRENT_YEAR = 18
+CURRENT_MONTH = 4
+
+while True:
+
+    cc_number = input("Enter cc number in format \'XXXX XXXX XXXX XXXX\': \n")
+    cc_number_list = cc_number.split(" ")
+    if len(cc_number_list) != 4:
+        print("Bad cc number format! Try again!")
+        continue
+    cc_number_str = ''
+    quartets_are_good = True
+    for quartet in cc_number_list:
+        if len(quartet) != 4:
+            print("Bad cc number format! Try again!")
+            quartets_are_good = False
+            break
+        try:
+            int(quartet)
+        except ValueError:
+            print("Bad cc number format! Try again now!")
+            quartets_are_good = False
+            break
+        cc_number_str += quartet
+    if not quartets_are_good:
+        continue
+
+    # проверка номера кредитной карты по алгоритму Луна!
```

Comment:
what kind of check is this one?

Reply:
Andrii was explaining using boolean as trigger at lesson on Saturday